pom.xml

```diff
@@ -926,13 +926,15 @@
                 <module>cli/ballerina-launcher</module>
                 <module>cli/ballerina-packerina</module>
 
+                <!--Should Run Before tests/ballerina-test-->
+                <module>misc/protobuf-ballerina</module>
```

Comment:
What about the entry in the release and default profiles?

Reply:
Fixed in 9e46733